apps/client/websocket/client.go

```diff
@@ -67,6 +67,11 @@ func (ws *WebsocketService) StartWSClient(ctx context.Context, workQueueChan cha
 					fmt.Printf("\n😒 Ignoring work request %s with difficulty %dx above our max %dx", serverMsg.Hash, serverMsg.DifficultyMultiplier, ws.maxDifficulty)
 					continue
 				}
+				if serverMsg.DifficultyMultiplier < minDifficulty {
+					fmt.Printf("\n😒 Ignoring work request %s with difficulty %dx below our min %dx", serverMsg.Hash, serverMsg.DifficultyMultiplier, minDifficulty)
+					continue
+				}
+
```

Comment:
It is not passed to the server, it is just like setting it in a constructor on a class, but struct since go doesn't have classes.

https://github.com/BananoCoin/boompow/blob/f18988f0f528a7a337c266d19a19efdd262c0af7/apps/client/websocket/client.go#L66

Reply:
And https://github.com/BananoCoin/boompow/blob/f18988f0f528a7a337c266d19a19efdd262c0af7/apps/client/websocket/client.go#L24

Reply:
Ok, thanks

Updated it to use the struct, commit f1ae276254de886801837c26dd57b53464bcb050